{"status": "broken", "description": "Checking if the projected position of the robot has not gone out of the boundary. \n        The boundary is the test lab navigation map and its dimensions are 10x10 m^2.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "nav_test"}, {"name": "subSuite", "value": "TestNavigation"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "5332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.nav_test"}], "stop": 1607974390603, "uuid": "81337d20-5732-4e3b-bc48-7b867c3fecb3", "historyId": "c3c08f87e17b0087ab8ae7b8508735bf", "name": "test_operation_zone_verification", "start": 1607974390603, "fullName": "tests.nav_test.TestNavigation#test_operation_zone_verification", "statusDetails": {"message": "NameError: global name 'hx' is not defined", "trace": "self = <tests.nav_test.TestNavigation instance at 0x7fd2139e3e60>\n\n    def test_operation_zone_verification(self):\n        \"\"\"Checking if the projected position of the robot has not gone out of the boundary.\n        The boundary is the test lab navigation map and its dimensions are 10x10 m^2.\n        \"\"\"\n    \n>       assert -5 <= hx <= 5\nE       NameError: global name 'hx' is not defined\n\ntests/nav_test.py:134: NameError"}, "attachments": [{"source": "d6d50bd3-0a79-495b-897c-b502abd2c86a-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "aa13fda5-2968-45d3-9f22-e917573953be-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "8e2e2ca7-1977-459a-8daa-ce8d92f85c2d-attachment.txt", "type": "text/plain", "name": "stderr"}]}